src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -1044,6 +1077,29 @@ - (void)setExpectedValues:(NSArray<NSDictionary<NSString *, id> *> *)values expe
     os_unfair_lock_unlock(&self->_lock);
 }
 
+- (void)removeExpectedValuesForAttributePath:(MTRAttributePath *)attributePath
+{
+    os_unfair_lock_lock(&self->_lock);
+    BOOL shouldReportValue;
+    NSDictionary<NSString *, id> * attributeValueToReport;
+    [self _setExpectedValue:nil
+                 attributePath:attributePath
+                expirationTime:nil
+             shouldReportValue:&shouldReportValue
+        attributeValueToReport:&attributeValueToReport];
+
+    MTR_LOG_INFO("%@ remove expected values for path %@ should report %@", self, attributePath, shouldReportValue ? @"YES" : @"NO");
```

Comment:
```suggestion
    MTR_LOG_INFO("%@ remove expected value for path %@ should report: %@", self, attributePath, shouldReportValue ? @"YES" : @"NO");
```

Reply:
Should be fixed as part of the latest changes